packages/zipkin-instrumentation-kafkajs/package.json

```diff
@@ -5,18 +5,19 @@
   "main": "lib/zipkin-instrumentation-kafkajs.js",
   "scripts": {
     "build": "babel src -d lib",
-    "pretest": "docker-compose up -d; timeout 60 bash -c 'until echo > /dev/tcp/localhost/9092; do sleep 1; done'",
-    "test": "mocha --require ../../test/helper.js",
+    "pretest": "(docker kill kafkajs-test; docker rm kafkajs-test) 2>&-; docker run -d -p 9092:9092 --name kafkajs-test --hostname localhost openzipkin/zipkin-kafka && timeout 60 bash -c 'until echo > /dev/tcp/localhost/9092; do sleep 1; done'",
```

Comment:
in local dev, tests would fail because the pre-existing docker containers weren't deleted. the result would be strange as it would appear parsing was busted because the same headers are always read... this was due to replaying whatever the first test message was. now this is fixed, but really we have to switch this to something else!